src/main/java/io/reactivex/Maybe.java

```diff
@@ -2892,6 +2892,31 @@ public final T blockingGet(T defaultValue) {
         ));
     }
 
+    /**
+     * Calls the shared consumer with the given onTerminate callback for each
```

Comment:
This sentence makes no sense. Please word it like the other similar operators: http://reactivex.io/RxJava/2.x/javadoc/io/reactivex/Completable.html#doOnTerminate-io.reactivex.functions.Action-

Reply:
Updated